examples/abitest/client/cpp/abitest.cc

```diff
@@ -195,25 +195,25 @@ bool run_aux_tests() {
   success &= check_page(&introspect, "/graph");
   success &= check_page(&introspect, "/objcount");
   success &= check_page(&introspect, "/node/0");
-  success &= check_page(&introspect, "/node/1");
+  success &= check_page(&introspect, "/node/3");
```

Comment:
Is this change still needed? Or does it now work with "/node/1".

Reply:
Aah, looks like it works with 1 now. I'm not sure what changed, do you have any ideas?

Reply:
I think that call to `proxy_for_new_node` in `AuxServer` was unnecessarily increasing the `node_id`.